client/app/pages/data-sources/show.js

```diff
@@ -1,20 +1,21 @@
 import { find } from 'lodash';
 import debug from 'debug';
 import template from './show.html';
+import notification from '@/services/notification';
 
 const logger = debug('redash:http');
 export const deleteConfirm = { class: 'btn-warning', title: 'Delete' };
-export function logAndToastrError(deleteObject, httpResponse, toastr) {
+export function logAndNotificateError(deleteObject, httpResponse) {
```

Comment:
```js
notification.error('Notificate isn\'t a word', 'Consider logAndNotifyError', { duration: null });
```

Reply:
It seems I've started to create new words :sweat_smile: 

![oops](https://user-images.githubusercontent.com/3356951/54678495-00ec5d00-4ae4-11e9-95e4-e8e1420bd96a.gif)